lib/bindConnectors/bindDynamicOutput.js

```diff
@@ -0,0 +1,31 @@
+var _                = require('lodash');
+var when             = require('when');
+var GenerateSchema = require('generate-schema');
+
+module.exports = function (message, options) {
+
+    return function (event) {
+
+        return when.promise(function (resolve, reject) {
+
+            when(message.dynamicOutput(event.body.input))
```

Comment:
Best confer with Chris first, because this brings into question why `bindTriggerRequest.js` has it but not `bindTriggerResponse.js` is different, and so it `bindMessage.js`